src/hooks/getInputDate.tsx

```diff
@@ -0,0 +1,11 @@
+export default function getInputDate() {
+  const today = new Date();
+
+  const year = today.getFullYear();
+  const month = ('0' + (today.getMonth() + 1)).slice(-2);
+  const day = ('0' + today.getDate()).slice(-2);
+
+  const dateString = year + '.' + month + '.' + day;
```

Comment:
템플릿 리터럴 사용하면 더 간단할 것 같아요!
```suggestion
  const dateString = `${year}.${month}.${day}`;
```